source/dubregistry/web.d

```diff
@@ -595,6 +632,29 @@ class DubRegistryFullWebFrontend : DubRegistryWebFrontend {
 		redirect("/my_packages/"~_packname);
 	}
 
+	@auth @path("/my_packages/:packname/set_logo")
+	void postSetLogo(scope HTTPServerRequest request, string _packname, User _user)
+	{
+		enforceUserPackage(_user, _packname);
```

Comment:
That would be an excellent use-case for either an `auth.Owner` role or a `@before` tag, but that's over the scope of this PR.